.changeset/poor-years-hang.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+docs: migrate `guide/types` snippets
```

Comment:
Just like [here](https://github.com/FuelLabs/fuels-ts/pull/3089#discussion_r1878869571), please add a task list of everything pending in the PR description.

Also, please include the CLI code in the PR and adjust the title accordingly.